@navikt/core/react/src/layout/base/BasePrimitive.tsx

```diff
@@ -142,12 +180,47 @@ export interface BasePrimitiveProps {
   flexGrow?: ResponsiveProp<string>;
 }
 
+export const BASE_PRIMITIVE_ALLOWED_PROPS: (keyof PrimtiveBaseProps)[] = [
```

Comment:
Need some help here: Any1 know a smart way to make sure `BASE_PRIMITIVE_ALLOWED_PROPS` includes every prop in `PrimtiveBaseProps`?

Reply:
Don't think it's possible without some complicated TS woodoo